apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/SpannerStressTests.cs

```diff
@@ -0,0 +1,133 @@
+﻿// Copyright 2017 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Google.Cloud.Spanner.V1.Logging;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Google.Cloud.Spanner.Data.IntegrationTests
+{
+    [Collection("Spanner Integration Tests")]
+    [PerformanceLog]
+    public class SpannerStressTests : BaseStressTest
+    {
+        private static int s_myCounter = 1;
+        private static readonly string s_guid = Guid.NewGuid().ToString();
+        private static readonly Random s_rnd = new Random(Environment.TickCount);
```

Comment:
`Random` cannot be used concurrently on multiple threads. Mark this with a `[ThreadStatic]` attribute so each gets its own copy and initialize it lazily (only the first thread to access the field will have its value created as it is right now).